L73-java-binderchannel/wireformat.md

```diff
@@ -35,7 +35,8 @@ For general transport lifecycle, using one of 1000 reserved transaction codes.
 *   Acknowledge bytes.
     *   Transaction code: BinderTransport.ACKNOWLEDGE_BYTES (3)
     *   From either transport to its peer's binder. Reports reception of
-        transaction data for flow control.
+        transaction data for flow control. The "num bytes" field should contain
+        the sum of "data size" of the parcels received until now.
```

Comment:
Worth adding a footnote link to Parcel.dataSize()?
https://developer.android.com/reference/android/os/Parcel#dataSize()

Reply:
Done.